Parser/Client/Browser.php

```diff
@@ -584,6 +624,21 @@ public function parse(): ?array
             $engineVersion = $browserFromUserAgent['engine_version'];
         }
 
+        $family  = self::getBrowserFamily((string) $short);
+        $appHash = $this->browserHints->parse();
+
+        if (null !== $appHash && $appHash['name'] !== $name) {
+            $name    = $appHash['name'];
+            $version = '';
+            $short   = '';
+
+            if (\preg_match('~Chrome\/.+ Safari\/537.36~i', $this->userAgent)) {
```

Comment:
Dont you need to escape the dot too to `if (\preg_match('~Chrome\/.+ Safari\/537\.36~i', $this->userAgent)) {`?